client/src/components/Common/Buttons/PrimaryButton/index.js

```diff
@@ -1,14 +1,22 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { theme1, theme2 } from "../../../../resources/style-constants";
+import { theme1, theme2, lightBlue, grey3 } from "../../../../resources/style-constants";
 import Button from "../Button";
 
+
 const theme = {
   default: theme1,
   hover: theme2,
   disabled: theme1
 };
 
+const accent = {
+    default: lightBlue,
+    hover: theme1,
+    disabled: grey3,
+    borderRadius: 0
+  };
+
 const PrimaryButton = ({
   onClick,
   fullWidth,
```

Comment:
So close to being done. you'll want to pull in the theme prop here. 
`
const PrimaryButton = ({
  onClick,
  fullWidth,
  disabled = true,
  type = "theme",
  children,
  ...props
}) => {
`
(Setting type = "theme" will default to theme if there isn't a type that is passed to the component. 

Then down below do something like:
`return (
    <Button theme={type === "theme" ? theme : accent }{...{ fullWidth,  disabled, onClick, ...props }}>
      {children}
    </Button>
  );
`